SyncedSideBar.py

```diff
@@ -124,10 +124,13 @@ def revealLater():
 
 class SideBarListener(sublime_plugin.EventListener):
     def on_activated(self, view):
+
+        self.packageSettings = sublime.load_settings('SyncedSideBar.sublime-settings')
+
         # don't even consider updating state if we don't have a window.
         # reveal in side bar is a window command only.
         # 'goto anything' activates views but doesn't set a window until the file is selected.
-        if not view.window():
+        if view.settings().get("is_widget", False):
```

Comment:
This change doesn't appear to be necessary anymore? The `focus_view` fix seems to be all that is required to resolve this issue.